### PR TITLE
[Do Not Merge] Omit selflinks from the SYNC message.

### PR DESCRIPTION
**Pull request by adamkingit**

### CODE REVIEW

File: src/razeedash/DelayedSendArray.js [hunk 1]
```diff
@@ -32,40 +32,41 @@ const _queueObject = Symbol('queueObject');
...
+    this._sendPromises = [];
+    if (this._trackSendPromises) {
+      this._pollStarted = new Date(Date.now());
     }
   }
 
   get maxItems() {
     return this._maxItems;
   }
+  get getSendPromises() {
```
Comment (adamkingit) [inline src/razeedash/DelayedSendArray.js]: get sendPromises() {

File: src/razeedash/DelayedSendArray.js [hunk 2]
```diff
@@ -74,62 +75,63 @@ module.exports = class DelayedSendArray {
...
-        'razee-org-key': process.env.RAZEEDASH_ORG_KEY
+        'razee-org-key': process.env.RAZEEDASH_ORG_KEY,
+        'poll-cycle': this._pollStarted
       },
       json: true,
       body: data,
 
       maxAttempts: options.maxAttempts || 5, // (default) try 5 times
-      retryDelay: options.retryDelay || 5000, // (default) wait for 5s before trying again
+      retryDelay: options.retryDelay || 3000, // (default) wait for 5s before trying again
```
Comment (adamkingit) [inline src/razeedash/DelayedSendArray.js]: update comment

File: src/razeedash/Sender.js [hunk 3]
```diff
@@ -46,37 +47,46 @@ module.exports = class RazeedashSender {
   }
 
 
-  reset(){
-    this.flush();
-    this._sentSelflinks={};
+  reset() {
```
Comment (adamkingit) [inline src/razeedash/Sender.js]: How do we avoid losing the objects in the outbound array when flush is called?

File: src/razeedash/DelayedSendArray.js (hunk 1 above)
Comment (alewitt2) [reply src/razeedash/DelayedSendArray.js]: i had that originally. But it looked like a "command" when you call it.. ie. `dsa.sendPromises`. so i figured it looks better as `dsa.getSendPromises`. What do you think?

File: src/razeedash/DelayedSendArray.js (hunk 2 above)
Comment (alewitt2) [reply src/razeedash/DelayedSendArray.js]: done

File: src/razeedash/Sender.js (hunk 3 above)
Comment (alewitt2) [reply src/razeedash/Sender.js]: im not sure i understand the question. We call flush (which send all resources in the outbound array and puts the send promise in `this._dsa.getSendPromises`), then we wait for the `this._dsa.getSendPromises` to finish here https://github.com/razee-io/Watch-keeper/pull/38/files#diff-02952b85e05587f619c17cc67efa16deR61

File: src/razeedash/DelayedSendArray.js (hunk 1 above)
Comment (adamkingit) [reply src/razeedash/DelayedSendArray.js]: I see your pt. Consider making it getSendPromises() but I am ok either way

**Pull request merged**